opentelemetry-sdk/tests/context/propagation/test_b3_format.py

```diff
@@ -35,22 +35,35 @@ def setUpClass(cls):
         cls.serialized_span_id = b3_format.format_span_id(
             trace.generate_span_id()
         )
+        cls.serialized_parent_span_id = b3_format.format_span_id(
+            trace.generate_span_id()
+        )
 
     def test_extract_multi_header(self):
```

Comment:
it feels to me that this test was sufficient by itself (since it exercises the base case of serialization / de-serialization), and have a separate test that verifies the parent_id propagation.

This test as is would also have been a good test for an empty parent.

Reply:
Sorry, I don't understand this comment, can you explain further, please?

Reply:
Yep definitely. What I was trying to say was: this test already should just work, and actually exercises a valuable case where the span has no parent.

So I was arguing to not modify this test case at all, and instead add another one that verifies that a parent_id field is set if a span has a parent.

Reply:
Sure, since you already approved these changes, @toumorokoshi, do you prefer if I open separate issues for your pending requests like this one or is if ok for you if I commit them directly in this PR again?